### ISSUE TITLE
Add Github upload support

### ISSUE THREAD

**Issue by slipb** [ondras/my-mind#107]
Currently the file only support upload to Google drive, which is pretty cool, 
Wish to support Github,Thanks! If the json file could by sync with Git, we can even have versin control~

**Comment by ondras** [comment]
Do they have an API for that? Keep in mind that "My Mind" is a client-side app running in a browser on a different domain/origin, so the options are a bit limited.

**Comment by slipb** [comment]
Dear Ondras，
Thanks for your reply! Im a beginner of programming, so I don't know too much detail about Github APIs.
https://www.draw.io/ if you take a look, I don't know how they achieve that.
And my vscode also use gist for sync setting by a jason file, Im not sure if its possible for "my mind"

**Comment by ondras** [comment]
> https://www.draw.io/ if you take a look, I don't know how they achieve that.

Yeah, apparently they provide the necessary infrastructure for this. This feature request makes sense then.

I do not currently have enough time to implement this, but my availability can change in the future. Also, PRs are welcome.
